Sources/Reader/Reader.entitlements

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>webcredentials:wordpress.com</string>
+		<string>webcredentials:*.wordpress.com</string>
+		<string>applinks:links.wp.a8cmail.com</string>
```

Comment:
This link appears to be used to open links in marketing emails. Maybe we should not fight with the WordPress and Jetpack apps to open those links?

Reply:
Followed up in https://github.com/wordpress-mobile/WordPress-iOS/pull/24487